sql/ddl_internal.sql

```diff
@@ -119,10 +119,18 @@ BEGIN
 
     IF num_slices IS NULL THEN
         -- Open dimension
-        IF column_type NOT IN ('BIGINT', 'INTEGER', 'SMALLINT', 'TIMESTAMP', 'TIMESTAMPTZ') THEN
+        IF column_type NOT IN ('BIGINT', 'INTEGER', 'SMALLINT', 'DATE', 'TIMESTAMP', 'TIMESTAMPTZ') THEN
             RAISE EXCEPTION 'illegal type for column "%": %', column_name, column_type
             USING ERRCODE = 'IO102';
         END IF;
+        IF column_type = 'DATE'::regtype AND interval_length < _timescaledb_internal.interval_to_usec('1 day') THEN
```

Comment:
Yeah but what I wanted to avoid is having to (a) explain the difference in units in docs and (b) have to know the type of the column in the table when debugging and seeing create_hypertable calls.